backend/apiviews/author_views.py

```diff
@@ -114,6 +114,19 @@ def get_current_user(self, request):
         else:
             return Response({"authenticated": False}, status=status.HTTP_401_UNAUTHORIZED)
 
+    def update_user(self, request, authorId, *args, **kwargs):
+        # author/update/<path:authorId>
```

Comment:
why do we need frontend to pass in author_id? This endpoint is only for currently authenticated user anyway, user information can be retrieved directly using `request.user`

Reply:
kk